plugins/identities/http.go

```diff
@@ -102,6 +117,14 @@ func (c *HTTPClientConfig) Decrypt() error {
 			}
 			c.Auth.Token = decryptedValue
 		}
+
+		if c.Auth.GoogleServiceAccountCredentials != "" {
+			decryptedValue, err := c.crypto.Decrypt(c.Auth.GoogleServiceAccountCredentials)
+			if err != nil {
+				return err
+			}
+			c.Auth.GoogleServiceAccountCredentials = decryptedValue
```

Comment:
Would this decrypted value here be base64 encoded  or json itself ?

Reply:
I intentionally make it expects a json (string) because apparently in policy service this config is parsed twice so the encrypted creds is successfully decoded again using base64 and that causes we storing the invalid creds